.editorconfig

```diff
@@ -8,6 +8,8 @@ root = true
 indent_style = tab
 end_of_line = lf
 charset = utf-8
+indent_style = tabs
```

Comment:
hm should indent_size be `tab` as well?

https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties#indent_size